spec/features/map/granule_selection_spec.rb

```diff
@@ -47,93 +47,87 @@
       nth_granule_list_item(10).click
     end
 
-    it "highlights the selected granule in the granule list" do
+    it 'highlights the selected granule in the granule list' do
       expect(granule_list).to have_selector('.panel-list-selected', count: 1)
     end
 
-   # EDSC-1395: Disabling this test for the moment as it is being flaky.
-   # This test is questionable as I do not see how the presence of
-   # 5 locators of '#map path' is evidence that the selected granule is highlighted.
-   # Further, this value seems to change from 5 to 3 then back to 5 almost
-   # randomly, making this test very flaky.  Re-evaluate at appropriate time.
-
-   # it "highlights the selected granule on the map" do
-   #   expect(page).to have_selector('#map path', count: 5)
-   # end
+    it 'highlights the selected granule on the map' do
+      expect(page).to have_selector('#map path', count: 5)
+    end
 
-    it "displays a link to remove the granule in the granule list" do
+    it 'displays a link to remove the granule in the granule list' do
       expect(granule_list).to have_link('Remove granule')
     end
 
-    it "displays a link to remove the granule on the map" do
+    it 'displays a link to remove the granule on the map' do
       expect(page).to have_selector('#map .panel-list-remove', count: 1)
     end
 
     it "displays the granule's temporal extents on the map" do
       expect(page).to have_selector('.granule-spatial-label-temporal', count: 1)
     end
 
-    it "displays the granule above all other granules" do
+    it 'displays the granule above all other granules' do
       synchronize do
         expect(page.execute_script(is_temporal_ordered_script)).to be_falsey
       end
     end
 
-    it "centers the map over the selected granule", pending_updates: true do
-      expect(page).to match_map_center(-9, 7)
+    it 'centers the map over the selected granule' do
+      expect(page).to match_map_center(-14.625, 0)
```

Comment:
Is there a known reason that the center changed?

Reply:
Not 100% sure, could be the new version of leaflet, I'm not sure how long this test had been pending